scripts/generate_notebooks.py

```diff
@@ -0,0 +1,57 @@
+"""A script to generate Notebooks for documentation."""
+import argparse
+import fnmatch
+import os
+import typing as t
+from pathlib import Path
+
+import requests
+from jupytext import cli as jupytext_cli
+
+BASE_EXAMPLE_URL = "https://docs.zhinst.com/zhinst-toolkit/en/latest/examples"
+EXAMPLES_DIR = Path(__file__).parent.parent / "examples"
+EXAMPLES_ONLY_SYNC = ["nodetree.md"]
+EXCLUDED_FILES = ["README.md"]
+
+
+def download_example_file(filename: str) -> bytes:
+    """Download example file."""
```

Comment:
It the non google style conform documentation intentional?

Reply:
Left them out as they are not part of the public API, but I have now added them.